packages/@okta/vuepress-site/docs/concepts/direct-authentication/index.md

```diff
@@ -0,0 +1,102 @@
+* **Traditional authentication**: Often involves multiple redirects to identity providers, which can disrupt the user experience and lead to longer sign-in times
+
+**Complexity**:
+
+* **Direct authentication**: Simplifies the authentication flow by eliminating the need for intermediary identity providers, making it easier to implement in certain contexts
+* **Traditional authentication**: Typically requires more complex setups, including managing multiple identity providers and handling various protocols
+
+**Control**:
+
+* **Direct authentication**: Offers developers greater control over the authentication process, allowing for customization of security measures and user interfaces
+* **Traditional Authentication**: Relies on third-party identity providers, which can limit control over the authentication experience and security practices
+
+**Security responsibility**:
+
+* **Direct authentication**: Places the responsibility of security on app developers, who must ensure secure handling of user credentials
+* **Traditional authentication**: Distributes security responsibilities across multiple identity providers, potentially reducing the burden on individual apps
+
+**Latency**:
+
+* **Direct authentication**: Reduces latency by eliminating the need for network hops associated with redirects to identity providers
+* **Traditional authentication**: Can introduce delays due to the multiple steps involved in redirecting to and from identity providers
+
+**Use cases**:
+
+* **Direct authentication**: Best suited for apps where speed and simplicity are critical, such as mobile apps and single-page apps
+* **Traditional authentication**: More appropriate for scenarios that require federated identity management and Single Sign-On (SSO) capabilities across multiple apps
+
+## Disadvantages and considerations
+
+* **Increased security responsibility**: The client app and backend service bear more direct responsibility for secure credential handling (storage, transmission, validation).
+* **Credential exposure risk**: If not implemented carefully, direct transmission of credentials can increase the risk of interception or compromise. Strong encryption (TLS/SSL) is mandatory.
+* **Complexity for federation**: Less suitable for scenarios requiring complex federated identity management across multiple distinct services or organizations.
+* **Lack of SSO (Out-of-the-Box**): Doesn't inherently provide SSO across disparate apps without more customization.
+* **Compliance**: May require careful consideration of compliance with various security and privacy regulations (for example, GDPR, HIPAA) regarding direct credential handling.
+
+## Security best practices
+
+Implementing direct authentication requires strict adherence to security best practices to mitigate inherent risks:
+
+* **Always use HTTPS/TLS**: Encrypt all communication between the client and Okta.
```

Comment:
Similar to above: this is true, but I think is true of all okta apis. I don't think we allow non-ssl requests